### PR TITLE
[16.0][MIG] account_fiscal_month: Migration to 16.0

### PR DESCRIPTION
**Pull request by BT-anieto**

### CODE REVIEW

File: account_fiscal_month/views/date_range_type.xml
Comment (BT-rmartin) [inline account_fiscal_month/views/date_range_type.xml:5]: Why not keeping same xml id as parent view_date_range_type_form_view and putting the id before the model?

File: account_fiscal_month/views/date_range_type.xml
Comment (BT-rmartin) [inline account_fiscal_month/views/date_range_type.xml:16]: Same comment as above

File: account_fiscal_month/__manifest__.py
Comment (BT-rmartin) [inline account_fiscal_month/__manifest__.py:12]: Better in several lines the depends and the data?

File: account_fiscal_month/models/date_range_type.py
Comment (BT-rmartin) [inline account_fiscal_month/models/date_range_type.py:11]: If it's readonly when is the the value assigned? Or the goal is just having the unique date.range.type created as business data as fiscal month?

File: account_fiscal_month/models/date_range_type.py
Comment (BT-rmartin) [inline account_fiscal_month/models/date_range_type.py:18]: Couldn't you better delete the others and skip the deletion of this one?

**Review by BT-rmartin — CHANGES_REQUESTED** [review]
Good job with the migration. Just some tiny improvements that you might consider

**Comment by BT-anieto** [comment]
@BT-rmartin As far as I know, all the changes you request should be addressed in another PR tagged with [IMP] after the migration. The migration should minimise the changes. The only 'big' change I've made is the setUp by setUpClass, since I have seen several times the change in other similar PRs.

If you are sure that your improvement can be addressed in this PR, I will do them because I agree with your suggestions

**Review by BT-rmartin — APPROVED** [review]
@BT-anieto You are right. Those changes should be done after merging probably. 
Regarding your change to setUpClass it's perfect, as now TransactionCase acts as the old SavepointCase and then as it's a common creation of data for all tests we will save some time in the execution

**Review by BT-nfrancisco — APPROVED** [review]

**Comment by BT-anieto** [comment]
@OCA/accounting-maintainers Could it be merged?

**Review by etobella — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**
